example/simple/Dockerfile

```diff
@@ -0,0 +1,16 @@
+
+FROM busybox
+
+RUN echo 1 > /a
+RUN echo 2 > /b
+VOLUME /bar
+RUN mkdir /c && echo 3 > /c/a && echo 4 > /c/b
+RUN rm /bin/ls
+RUN rm /b
+EXPOSE 8080
+RUN rm /c/b
+VOLUME /baz
+EXPOSE 3000
+# RUN echo > /bin/hostname
+RUN rm /bin/ping
+# RUN rm /bin/hostname
```

Comment:
I think it'd be good to add some testing for deleting a file then re-adding it later; as this is where docker-squash runs into problems, I'd like to make sure we're not vulnerable to it as well